management/azure_mgmt_analysis_services/azure_mgmt_analysis_services.gemspec

```diff
@@ -5,8 +5,8 @@
 lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
-require 'generated/azure_mgmt_analysis_services/module_definition'
-require 'generated/azure_mgmt_analysis_services/version'
+require_relative 'lib/module_definition'
```

Comment:
thanks for the pointer, I think I'll stick with "require" for now to be consistent, and open an issue to tackle "require_relative" and loading enhancement, mainly because 1) autorest generator would need to be re-adjusted, 2) not sure if the LOAD_PATHs we have are appropriate or need to be adjusted as well. Does this sound good?